prerendering.bs

```diff
@@ -787,6 +802,30 @@ Add {{[DelayWhilePrerendering]}} to {{Navigator/getUserMedia()}}, {{MediaDevices
   1. If [=this=]'s [=relevant global object=]'s [=Window/navigable=] is a [=prerendering navigable=], then return.
 </div>
 
+<h4 id="web-audio-patch">Web Audio API</h4>
+
+The concept, [=allowed to start=], is used in the spec, but details are not defined as implementation details.
+To restrict auto playback while prerendering, add the following rule in the [=AudioContext=] interface section.
+
+The [=AudioContext=] is never [=allowed to start=] while prerendering.
+
+Also modify [=AudioContext constructor=] steps to have the following step before returning the constructed object.
+
+1. Else if the context is not [=allow to start=] only due to the prerendering, then append the following steps to [=this=]'s [=Document/post-prerendering activation steps list=].
```

Comment:
What does it mean by "Else if" here?

Reply:
Yes.

Reply:
@hoch Are you ok with the current PR, or do you think we need some editing here to clarify the patch's context a little more?

Reply:
The current PR looks good to me. Thanks for asking!